tests/integration/test_pairwise_significance.py

```diff
@@ -324,3 +360,325 @@ def test_mr_x_mr_pairwise_t_tests(self):
         np.testing.assert_array_almost_equal(
             actual.p_vals, load_python_expression("mr-x-mr-pw-pvals")
         )
+        # test new api also
+        np.testing.assert_array_almost_equal(
+            slice_.pairwise_significance_t_stats[1],
+            load_python_expression("mr-x-mr-pw-tstats"),
+        )
+        np.testing.assert_array_almost_equal(
+            slice_.pairwise_significance_p_vals[1],
+            load_python_expression("mr-x-mr-pw-pvals"),
+        )
+
+
+class TestOverlapsCounts(TestCase):
```

Comment:
General thought is that this class maybe doesn't belong in this test module, here we are testing the cubepart interface but in this class you are testing the assembler basically ... i would move this class in the right test module

Reply:
I agree, but it does test the numbers rather than the structure. And I haven't really found a good place for it other than the current place. I have however added one test that tests the structure (and not numbers), and placed it in a different module.

Reply:
ok, figured out how to refactor